src/System.Diagnostics.Process/src/System/Diagnostics/Process.WinRT.cs

```diff
@@ -13,18 +13,18 @@ private bool StartCore(ProcessStartInfo startInfo)
                 : StartWithCreateProcess(startInfo);
         }
 
-        private string GetMainWindowTitle() => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        private string GetMainWindowTitle() => string.Empty;
 
-        public IntPtr MainWindowHandle => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        public IntPtr MainWindowHandle => IntPtr.Zero;
 
-        private bool CloseMainWindowCore() => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        private bool CloseMainWindowCore() => false;
 
-        public string MainWindowTitle => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        public string MainWindowTitle => string.Empty;
 
-        private bool IsRespondingCore() => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        private bool IsRespondingCore() => true;
 
-        public bool Responding => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        public bool Responding => true;
 
-        private bool WaitForInputIdleCore(int milliseconds) => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        private bool WaitForInputIdleCore(int milliseconds) => false;
```

Comment:
Would true be a better default? false means it's not idle, and calling code is likely to wait again and again. Whreas the truth is that it MIGHT be idle so better to return true and calling code may continue happily.